front/src/plugins/kakao-map.js

```diff
@@ -29,8 +29,18 @@ const KakaoMap = {
         level: 2,
       };
       this.map = new kakao.maps.Map(mapContainer, options);
+      this.clusterer = createClusterer.call(this);
```

Comment:
```suggestion
      this.clusterer = createClusterer();
```

const로 선언한다면 같은 this를 공유하게 되어서 call을 빼도 괜찮아요!

Reply:
오 되는군요!